pyproject.toml

```
@@ -28,7 +28,7 @@ ignore_missing_imports = true
 
 [tool.poetry.dependencies]
 python = "^3.8.10"
-pydantic = "2.6.1"
+pydantic = "^2.8.2"
```

```suggestion
pydantic = "^2.6.1"
```

This should be fine as well no?

---

It'll by default install the newest version of pydantic then!